src/istgtcontrol.c

```diff
@@ -912,9 +914,31 @@ exec_command(UCTL_Ptr uctl)
 	int rc = 0;
 	char *name = uctl->setargv[0];
 	char *value = uctl->setargv[1];
-
-	uctl_snprintf(uctl, "%s \"%s\" \"%s\" \n",
-		uctl->cmd, name, value);
+	int i = 0, listcnt;
+	char *id_list = NULL;
+
+	listcnt = uctl->setargcnt - 2;
+	if (listcnt != 0) {
```

Comment:
This should be `if (listcnt > 0)`.

Reply:
Definitely It will not be < 0 but anyhow to be specific I will update with `listcnt > 0`

Reply:
handled